docs/source/get-started/suggested-models.md

```diff
@@ -0,0 +1,173 @@
+# Suggested Models
+
+In theory, Lumigator supports any model that is uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending)
+and is trained for *summarization*. However, we have extensively tested a few models and created
+an endpoint to easily retrieve them.
+
+In this guide, we assume that you have already [installed Lumigator locally](quickstart), and have a
+running instance. To get a list of suggested models, you can use the following command:
+
+::::{tab-set}
+
+:::{tab-item} cURL
+:sync: tab1
+
+```console
+user@host:~/lumigator$ curl -s http://localhost:8000/api/v1/models/summarization | jq
+{
+  "total": 9,
+  "items": [
+    {
+      "name": "facebook/bart-large-cnn",
+      "uri": "hf://facebook/bart-large-cnn",
+      "description": "BART is a large-sized model fine-tuned on the CNN Daily Mail dataset.",
+      "info": {
+        "parameters_count": "406M",
+        "tensor_type": "F32",
+        "model_size": "1.63GB"
+      },
+      "tasks": [
+        {
+          "summarization": {
+...
+```
+
+:::
+
+:::{tab-item} Python SDK
+:sync: tab2
+```python
+from lumigator_sdk.lumigator import LumigatorClient
+
+lm_client = LumigatorClient("localhost:8000")
+lm_client.models.get_suggested_models("summarization")
+```
+:::
+
+::::
+
+The output will show a list of suggested models we have tested. The `uri` field is the one you
+should use when creating a new evaluation job. The response also includes other useful information,
+such the model size and the default parameters used for evaluation. These fields are not applicable
+to every model, but they are included for the ones we have tested.
+
+## Model Types and Parameters
+
+The following table shows the models we have tested and their respective types.
+The `via HuggingFace` column indicates if the model is available via the Hugging Face Hub. The
+`via API` column indicates if the model is available via an external API. The `llamafile` column
+indicates if the model is distributed as a [llamafile](https://github.com/Mozilla-Ocho/llamafile).
```

Comment:
Would it make sense to add something to the effect of...?
```suggestion
indicates if the model is distributed as a [llamafile](https://github.com/Mozilla-Ocho/llamafile). Please note we do not at present launch a llamafile for you, Lumigator currently assumes you have already launched it.
```

Reply:
I will add it as an admonition, so it gets more attention.